handsontable/src/plugins/manualColumnResize/manualColumnResize.js

```diff
@@ -393,7 +393,9 @@ export class ManualColumnResize extends BasePlugin {
    * @returns {boolean}
    */
   checkIfColumnHeader(element) {
-    return !!closest(element, ['THEAD'], this.hot.rootElement);
+    const thead = closest(element, ['THEAD'], this.hot.rootElement);
+
+    return thead === this.hot.view._wt.wtOverlays.topOverlay.clone.wtTable.THEAD;
```

Comment:
This condition makes the resize handle not appear when the column headers are in the corner overlay (overlapping fixed rows and columns)

Reply:
Thanks for spotting that. I fixed and covered the change within https://github.com/handsontable/handsontable/pull/10650/commits/e8be061f7b55a3c90e058a284dadd9e3a3c9fa59.